api/agentpb/agent.proto

```diff
@@ -374,12 +374,24 @@ message StartJobRequest {
       S3LocationConfig s3_config = 10;
     }
   }
+  // MySQLBackupRestore is job for MySQL backup restore service.
+  message MySQLBackupRestore {
```

Comment:
Not sure, but maybe `MySQLRestore` is a better name.

Reply:
or `MySQLRestoreBackup`, because on API level we use `RestoreBackup`

Reply:
MySQLRestore seems better, @Dasio what are your thoughts?

Reply:
I would say `MySQLRestoreBackup`.

Reply:
To me, `MySQLRestoreBackup` is more clear, than the other way round ) I just reads better and avoids ambiguity

Reply:
thanks, naming changed